app/repositories/GMPBulkSessionRepository.scala

```diff
@@ -35,7 +35,8 @@ import scala.concurrent.{ExecutionContext, Future}
 class GMPBulkSessionRepository @Inject() (
                                            mongoComponent: MongoComponent,
                                            appConfig: ApplicationConfig,
-                                           implicit val encryption: Encryption
+                                           implicit val encryption: Encryption,
+                                           clock:Clock
```

Comment:
Will just need a bit of clarification on how this works? Can have a little chat this afternoon, but obviously sort out other comment for now!